tests/e2e/specs/setup-mc/step-4-complete-campaign.test.js

```diff
@@ -443,4 +443,28 @@ test.describe( 'Complete your campaign', () => {
 			}
 		} );
 	} );
+
+	test.describe( 'Free Ad Credit', () => {
+		test( 'should not see the Free Ad Credit section if the account is not eligible', async () => {
+			await setupAdsAccountPage.mockAdsAccountDisconnected();
+			await completeCampaign.goto();
+
+			await expect(
+				page.getByText(
+					'Spend $500 to get $500 in Google Ads credits!'
+				)
+			).not.toBeVisible();
+		} );
```

Comment:
This test case doesn't fulfill its test purpose:

- The Google Ads account should have been connected in this step. Mocking it with disconnected status is not ideal.
- It incorrectly passes the assertion while the page is still in loading state
   ![image](https://github.com/user-attachments/assets/bc6144ab-00b0-4a43-b9ba-7f4d0adac50e)

Reply:
👍